From 1eb5169a7820869ee7c193c0935a36e0d6774213 Mon Sep 17 00:00:00 2001
From: Yotam Ben Dosa <ybendosa@habana.ai>
Date: Thu, 26 Dec 2019 15:51:17 +0200
Subject: [PATCH] [SW-0] make: hls: add missing separator

The environment deployment command was mixed with the variable
initialization for the the devtool wrappers, a semicolon is required in
order to separate them.

Change-Id: Ic1f404cce33190addb5c6bda4c4ddbcec77dd1a7
Signed-off-by: Yotam Ben Dosa <ybendosa@habana.ai>
---

diff --git a/scripts/make/hls/Makefile b/scripts/make/hls/Makefile
index 64afae3..5e25c95 100644
--- a/scripts/make/hls/Makefile
+++ b/scripts/make/hls/Makefile
@@ -131,35 +131,39 @@
 
 openbmc-add:
 	@cd $(OPENBMC_DEPLOY_DIR);					\
-	$(OPENBMC_ENV) $(OPENBMC_BITBAKE_ENV)				\
-	devtool add "$(URI)"
+	$(OPENBMC_ENV);							\
+	$(OPENBMC_BITBAKE_ENV) devtool add "$(URI)"
 
 openbmc-%-modify:
 	@cd $(OPENBMC_DEPLOY_DIR);					\
-	$(OPENBMC_ENV) $(OPENBMC_BITBAKE_ENV)				\
-	devtool modify $(patsubst openbmc-%-modify,%,$@);		\
-	terminator							\
+	$(OPENBMC_ENV);							\
+	$(OPENBMC_BITBAKE_ENV) devtool modify				\
+		$(patsubst openbmc-%-modify,%,$@); terminator		\
 	--working-directory="$(OPENBMC_DEVTOOL_SOURCES)/$(patsubst	\
 	openbmc-%-modify,%,$@)" &
 
 openbmc-%-menuconfig:
 	@cd $(OPENBMC_DEPLOY_DIR);					\
-	$(OPENBMC_ENV) $(OPENBMC_BITBAKE_ENV)				\
+	$(OPENBMC_ENV);							\
+	$(OPENBMC_BITBAKE_ENV)						\
 	bitbake -c menuconfig $(patsubst openbmc-%-menuconfig,%,$@)
 
 openbmc-%-build:
 	@cd $(OPENBMC_DEPLOY_DIR);					\
-	$(OPENBMC_ENV) $(OPENBMC_BITBAKE_ENV)				\
+	$(OPENBMC_ENV);							\
+	$(OPENBMC_BITBAKE_ENV)						\
 	devtool build $(patsubst openbmc-%-build,%,$@)
 
 openbmc-%-clean:
 	@cd $(OPENBMC_DEPLOY_DIR);					\
-	$(OPENBMC_ENV) $(OPENBMC_BITBAKE_ENV)				\
+	$(OPENBMC_ENV);							\
+	$(OPENBMC_BITBAKE_ENV)						\
 	bitbake -c clean $(patsubst openbmc-%-clean,%,$@)
 
 openbmc-%-cleanall:
 	@cd $(OPENBMC_DEPLOY_DIR);					\
-	$(OPENBMC_ENV) $(OPENBMC_BITBAKE_ENV)				\
+	$(OPENBMC_ENV);							\
+	$(OPENBMC_BITBAKE_ENV)						\
 	bitbake -c cleanall $(patsubst openbmc-%-cleanall,%,$@)
 
 openbmc-%-remove:
